In Boost 1.88, `boost::process` was switched from defaulting to
`boost::process::v1` to `boost::process::v2`.  The `::v1` header file which
included various common headers was also removed at this time, meaning each
header file has to be included separately.
--
diff --git a/common/ChildProcess/ChildProcess.cpp b/common/ChildProcess/ChildProcess.cpp
index 1974cda..7295dac 100644
--- a/common/ChildProcess/ChildProcess.cpp
+++ b/common/ChildProcess/ChildProcess.cpp
@@ -34,8 +34,14 @@
 
 #include <atomic>
 #include <boost/asio.hpp>
-#include <boost/process.hpp>
-#include <boost/process/extend.hpp>
+#include <boost/process/v1/args.hpp>
+#include <boost/process/v1/async_pipe.hpp>
+#include <boost/process/v1/child.hpp>
+#include <boost/process/v1/env.hpp>
+#include <boost/process/v1/environment.hpp>
+#include <boost/process/v1/extend.hpp>
+#include <boost/process/v1/io.hpp>
+#include <boost/process/v1/system.hpp>
 #include <thread>
 
 
@@ -140,7 +146,7 @@ struct ChildProcess::Impl
     }
 
     static auto StdLinesProcess(ChildProcess::Impl &self,
-                                boost::process::async_pipe &sourcePipe,
+                                boost::process::v1::async_pipe &sourcePipe,
                                 StdLines &target) -> boost::asio::awaitable<int>
     {
         auto cancellationState = co_await boost::asio::this_coro::cancellation_state;
@@ -183,7 +189,7 @@ struct ChildProcess::Impl
 
     void Run()
     {
-        namespace bp = boost::process;
+        namespace bp = boost::process::v1;
         process      = bp::child { executable,
                               bp::args(args),
                               bp::std_out > stdOutPipe,
@@ -381,17 +387,17 @@ struct ChildProcess::Impl
     boost::asio::io_context ioContext;
 
     Pipe fdResponses;
-    boost::process::async_pipe stdOutPipe;
-    boost::process::async_pipe stdErrPipe;
+    boost::process::v1::async_pipe stdOutPipe;
+    boost::process::v1::async_pipe stdErrPipe;
 
     FramedChannel fdChannel;
 
-    boost::process::child process;
+    boost::process::v1::child process;
 
     boost::filesystem::path executable;
     std::vector<std::string> args;
     std::optional<std::string> userName;
-    boost::process::environment environment;
+    boost::process::v1::environment environment;
 
     StdLines stdOutLines;
     StdLines stdErrLines;
@@ -448,7 +454,7 @@ ChildProcess Create(boost::filesystem::path const &executable,
     impl->channelFd   = channelFd;
     impl->args        = args;
     impl->userName    = userName;
-    impl->environment = static_cast<boost::process::environment>(boost::this_process::environment());
+    impl->environment = static_cast<boost::process::v1::environment>(boost::this_process::environment());
     for (auto const &pair : env)
     {
         impl->environment.emplace(pair.first, pair.second);
